neat_cache/lib/src/providers/resp.dart

```diff
@@ -177,7 +177,7 @@ class RespClient {
     final c = Completer<Object?>();
     _pending.addLast(c);
     try {
-      await _output.addStream(Stream.value(out.toBytes()));
+      _output.add(out.toBytes());
```

Comment:
I think it might be better to use a future to ensure that `_output.addStream` is called sequentially..

The problem with not awaiting it that if sending to the server fails, the exception won't cause the `RespClient` to close, hence, it might keep accepting commands.

Technically, I hope the stream will at some point enter a state where it throw exceptions synchronously, but without knowing the underlying implementation I'm not that will happen.